### ISSUE TITLE
Mind map nodes are not converted

### ISSUE THREAD

**Issue by augustoschnorr** [YashBhalodi/cardsy#1]
Hi!

I tried Cardsy to convert a mind map, but it does not work. I have a few other objects (text and code block) and those are converted to cards, but the mind map nodes are ignored. Is there a way to convert the mind map nodes as well?

Thanks!

**Comment by YashBhalodi** [comment]
Hey @Alcsaw! 

First of all, thanks for using Cardsy!

Regarding Mindmap, Yeah, I am aware that Cardsy is not generating cards from mindmap. After upgrading to MIRO SDK 2.0, I came to know that they still do not support reading mindmap nodes.

You can refer to official documentation [here](https://developers.miro.com/docs/miro-web-sdk-introduction#what-can-i-do-with-the-web-sdk).

You can vote for this feature request from MIRO on their roadmap [here](https://portal.productboard.com/bs3msuyjdbfr8dfxd84wngpn/c/39-crud-mindmap?utm_medium=social&utm_source=portal_share).

**Comment by augustoschnorr** [comment]
Hi Yash!

I thank you for creating it!

I understand. That's too bad, I though this tool would save me from a lot of rework 😅 
Well, I upvoted the feature request on the link you posted. Thanks for answering, Yash!

**Comment by Delfana** [comment]
Found my way here when testing out Cardsy for the first time today. I was looking forward to make my work with genealogy trees a bit easier. I have to manually recreate everything if I want to change a simple MindMap tree to a Card tree that can hold more information. Oh well 🤷‍♀️😊😇

I made a comment on the feature request too. I hope that you are able to and still interested in updating the app in the future 🙂

Happy Holidays!
